### PR TITLE
Adding contributing guidelines

### PR DESCRIPTION
**Pull request by bitschmidty**
This PR adds contributing guidelines for the compatibility matrix as well as the newsletters.

### CODE REVIEW

**Comment by harding** [comment]
Untested LGTM.  One suggestion would be to add `CONTRIBUTING.md` to the `excludes:` array in `_config.yml` so this page doesn't get copied into the deployed site (where its relative links won't work).

**Comment by bitschmidty** [comment]
@harding thanks for pointing that out. added

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+## Newsletter
+
+The newsletter is published weekly, on Wednesdays, around 15:00 UTC. Typically a
+pull request for the newsletter is opened the Saturday before publishing. Any
+review of the newsletter PRs is appreciated. However, feedback received after
+Tuesday UTC may not be incorporated due to time contraints.
```
Comment (adamjonas) [inline CONTRIBUTING.md]: contraints -> constraints

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Compatibility Data
+
+The compatibility section of the website is based off of
+[YAML](https://yaml.org/) files located in [_data/compatibility/](_data/compatibility/).
+The the compatibility images (usability screenshots, logos) are located in
```
Comment (adamjonas) [inline CONTRIBUTING.md]: The ~the~ compatibility

File: CONTRIBUTING.md (hunk 1 above)
Comment (bitschmidty) [reply CONTRIBUTING.md]: Thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Newsletter
+
+The newsletter is published weekly, on Wednesdays, around 15:00 UTC. Typically a
+pull request for the newsletter is opened the Saturday before publishing. Any
+review of the newsletter PRs is appreciated. However, feedback received after
+Tuesday UTC may not be incorporated due to time contraints.
+
+## Compatibility Data
+
+The compatibility section of the website is based off of
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply CONTRIBUTING.md]: changed

**Comment by bitschmidty** [comment]
Pushed commits with changes based on @adamjonas and @jnewbery feedback (cheers).

File: en/compatibility.md [hunk 4]
```diff
@@ -11,6 +11,11 @@ th, td { text-align: center; }
 h1, h2, h3, h4, h5, h6 { text-align: center; }
 </style>
 
+The compatibility dashboard displays evaluation of different Bitcoin wallets
+adoption of scaling technologies. If you are interested in contributing, please review the
+[contibuting guidelines](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md).
```
Comment (jonatack) [inline en/compatibility.md]: s/[contibuting/[contributing/

File: en/compatibility.md [hunk 5]
```diff
@@ -11,6 +11,11 @@ th, td { text-align: center; }
 h1, h2, h3, h4, h5, h6 { text-align: center; }
 </style>
 
+The compatibility dashboard displays evaluation of different Bitcoin wallets
```
Comment (jonatack) [inline en/compatibility.md]: - it looks like "evaluation" here should be "evaluations" or "an evaluation"

- perhaps s/different/various/

- s/wallets/wallets'/

File: README.md [hunk 6]
```diff
@@ -110,3 +110,8 @@ directory.  The following alternative options are available:
 
     ## After you build the site, you can run all of the tests (may take awhile)
     make test
+
+## Contributing
+
+If you would like to contribute to the Bitcoin Optech project, review the
```
Comment (jonatack) [inline README.md]: s/review/please review/?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+## Compatibility Data
+
+The compatibility section of the website is built from
+[YAML](https://yaml.org/) files located in [_data/compatibility/](_data/compatibility/).
+The compatibility images (usability screenshots, logos) are located in
+[img/compatibility/](img/compatibility/) with sub folders for each
+wallet/service. These files are free for anyone to repurpose/republish
+elsewhere.
+
+We welcome pull requests to the compatibility section of the site including
```
Comment (jonatack) [inline CONTRIBUTING.md]: s/site/site,/

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+testing the latest versions of previously tested services/wallets, adding notable
+usability screenshots, or adding of new service/wallet tests.
```
Comment (jonatack) [inline CONTRIBUTING.md]: perhaps s/adding of new/adding new/, like "adding notable" just before

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+## Newsletter
+
+The newsletter is published weekly, on Wednesdays, around 15:00 UTC. Typically a
+pull request for the newsletter is opened the Saturday before publishing. Any
+review of the newsletter PRs is appreciated. However, feedback received after
```
Comment (jonatack) [inline CONTRIBUTING.md]: ;)

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+Tuesday UTC may not be incorporated due to time constraints.
+
+## Compatibility Data
+
+The compatibility section of the website is built from
+[YAML](https://yaml.org/) files located in [_data/compatibility/](_data/compatibility/).
+The compatibility images (usability screenshots, logos) are located in
+[img/compatibility/](img/compatibility/) with sub folders for each
```
Comment (jonatack) [inline CONTRIBUTING.md]: s/sub folders/sub-folders/

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+wallet/service. These files are free for anyone to repurpose/republish
```
Comment (jonatack) [inline CONTRIBUTING.md]: perhaps "wallet or service."

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+testing the latest versions of previously tested services/wallets, adding notable
+usability screenshots, or adding of new service/wallet tests.
+
+When contributing changes to the compatibility data files, review and adhere to
+the YAML schema located in [_data/schemas/compatibility.yaml](_data/schemas/compatibility.yaml).
+
+If you believe any of the data in the compatibility section is incorrect, you
+can also [submit an issue](../../issues/) detailing what is wrong and how to correct it.
+
+If you want to request a new service/wallet be evaluated, or a new test that you
```
Comment (jonatack) [inline CONTRIBUTING.md]: perhaps "wallet or service"

File: README.md [hunk 13]
```diff
@@ -18,7 +18,7 @@ founding sponsors and contributions from member companies.
 If you're an engineer or manager at a Bitcoin company or an open source contributor and you'd like to be a part of this, please
 contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
 
-#### Building The Site Locally
+## Building The Site Locally
```
Comment (jonatack) [inline README.md]: s/The/the/

File: CONTRIBUTING.md (hunk 7 above)
Comment (jonatack) [reply CONTRIBUTING.md]: maybe s/to/for/

**Comment by jnewbery** [comment]
Added a commit moving the contributing note to the bottom of the compatibility overview page. Let me know what you think @bitschmidty

**Comment by bitschmidty** [comment]
> Added a commit moving the contributing note to the bottom of the compatibility overview page. Let me know what you think @bitschmidty

Yes, I prefer that as well.

**Pull request merged**